zk.js/src/utxo/select-in-utxos.ts

```diff
@@ -212,8 +217,9 @@ export function selectInUtxos({
   }
 
   // if mint is provided filter for only utxos that contain the mint
-  let filteredUtxos: Utxo[];
+  let filteredUtxos;
   let sumOutSpl = publicAmountSpl ? publicAmountSpl : BN_0;
+  /// TODO: review why we're using getUtxoArrayAmount which seems to be for inUtxos
```

Comment:
it's a utility function works for both in and out utxos equally since amount arrays are the same.